config/.editorconfig

```diff
@@ -0,0 +1,32 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.hbs]
+insert_final_newline = false
+
+[*.{js,ts,jsx,yml}]
+indent_size = 2
+
+[*.{diff,md}]
+trim_trailing_whitespace = false
+
+[*.go]
+indent_style = tab
```

Comment:
I was trying to look for a way to deactivate editorconfig totally for go. Haven't found anything :$